assets/wizards/site-design/views/main/index.js

```diff
@@ -0,0 +1,275 @@
+								className="mt2"
+								label={ __( 'Background color' ) }
+								color={ mods.header_color_hex }
+								onChange={ updateMods( 'header_color_hex' ) }
+							/>
+						) }
+					</div>
+				</div>
+				<div>
+					<ImageUpload
+						className="mt0"
+						style={ {
+							height: '96px',
+							...( mods.header_solid_background
+								? {
+										backgroundColor:
+											mods.header_color === 'custom'
+												? mods.header_color_hex
+												: mods.primary_color_hex,
+								  }
+								: {} ),
+						} }
+						label={ __( 'Logo', 'newspack' ) }
+						image={ mods.custom_logo }
+						onChange={ updateMods( 'custom_logo' ) }
+					/>
+					{ mods.custom_logo && (
+						<>
+							<SelectControl
+								className="mv0 dib"
+								label={ __( 'Size', 'newspack' ) }
+								value={ parseLogoSize( mods.logo_size ) }
+								onChange={ updateMods( 'logo_size' ) }
+								buttonOptions={ LOGO_SIZE_OPTIONS }
+							/>
+							<ToggleControl
+								checked={ mods.header_text }
+								onChange={ updateMods( 'header_text' ) }
+								label={ __( 'Display Site Title', 'newspack' ) }
+							/>
+							<ToggleControl
+								checked={ mods.header_display_tagline }
+								onChange={ updateMods( 'header_display_tagline' ) }
+								label={ __( 'Display Tagline', 'newspack' ) }
+							/>
+						</>
+					) }
+				</div>
+			</Grid>
+			<SectionHeader
+				title={ __( 'Footer', 'newspack' ) }
+				description={ __( 'Personalize the footer of your site', 'newspack' ) }
+			/>
+			<Grid gutter={ 32 }>
+				<div>
+					<TextControl
+						className="mt0"
+						label={ __( 'Copyright information', 'newspack' ) }
+						help={ __(
+							'Add custom text to be displayed next to a copyright symbol and current year in the footer. By default, it will display your site title.',
+							'newspack'
+						) }
+						value={ mods.footer_copyright || '' }
+						onChange={ updateMods( 'footer_copyright' ) }
+					/>
+					<ToggleControl
+						className="mv0"
+						checked={ mods.footer_color !== 'default' }
+						onChange={ checked => updateMods( 'footer_color' )( checked ? 'custom' : 'default' ) }
+						label={ __( 'Apply a background color to the footer', 'newspack' ) }
+					/>
+					{ mods.footer_color === 'custom' && (
+						<ColorPicker
+							className="mt2"
+							label={ __( 'Background color' ) }
+							color={ mods.footer_color_hex }
+							onChange={ updateMods( 'footer_color_hex' ) }
+						/>
+					) }
+				</div>
+				<ImageUpload
+					label={ __( 'Alternative Logo', 'newspack' ) }
+					info={ __( 'Optional alternative logo to be displayed in the footer.', 'newspack' ) }
+					style={ {
+						height: '96px',
+						...( mods.footer_color === 'custom' && mods.footer_color_hex
+							? { backgroundColor: mods.footer_color_hex }
+							: {} ),
+					} }
+					image={ mods.newspack_footer_logo }
+					onChange={ updateMods( 'newspack_footer_logo' ) }
+				/>
+			</Grid>
+			{ hasPreview && (
```

Comment:
This "floating" preview button looked a bit lost and bolted on to me. I had a hard time finding at first, actually:

<img width="2672" alt="Screen Shot 2021-03-05 at 2 50 09 PM" src="https://user-images.githubusercontent.com/2230142/110177381-1c7ccb00-7dc2-11eb-87ab-3027ea5ff15d.png">

Could we consider moving this to a simpler secondary-style button right next to the "Finish" button instead?

Reply:
No because it'd mean having to scroll all the way down to preview the site.

v2 with the iframe will give a much better visibility but I'm ok with the way it is right now for v1.

Having said that, I might play around with a full-width floating button.